examples/kubernetes/1.10/cilium-cm.yaml

```diff
@@ -26,8 +26,12 @@ data:
   # If you want to run cilium in debug mode change this value to true
   debug: "false"
   disable-ipv4: "false"
-  # If you want to clean cilium state; change this value to true
+  # If you want to clean cilium state; change this value to true.
+  # This may disrupt IP connectivity for existing pods.
   clean-cilium-state: "false"
+  # If you want to clean cilium BPF state; change this value to true
```

Comment:
If I were an external user looking at this, I wouldn't understand the difference between 'clean Cilium state' and 'clean cilium BPF state'  providing more insight as to what these values do (and how they are different from one another) here would be useful to ensure that users don't get confused.

Reply:
Yeah, this is a fair comment. I was trying to think how to explain this.

`clean-cilium-state`: Removes all persistent state during pod restart. Endpoints will not be restored, so they may receive new IP addresses upon restart. This also triggers `clean-cilium-bpf-state`.

`clean-cilium-bpf-state`: Removes all BPF maps from the filesystem. Upon restart, endpoints are restored with the same IP addresses, however any ongoing connections may be disrupted briefly.  Loadbalancing decisions will be reset, so any ongoing connections via a service may be loadbalanced to a different backend after restart.

Thoughts?

Reply:
/cc @tgraf

Reply:
Makes sense

Reply:
Done.